### PR TITLE
feat(create): accommodation creation with step and uploading media

### PR DESCRIPTION
**Pull request by melievkh**

### CODE REVIEW

File: src/axios/axiosInstance.ts [hunk 1]
```diff
@@ -24,7 +25,7 @@ axiosInstance.interceptors.request.use(
       config.headers.Authorization = `Bearer ${accessToken}`;
     }
 
-    if (config.url && config.url.includes(ENDPOINTS.signout) && refreshToken) {
+    if (config.url && config.url === ENDPOINTS.signout && refreshToken) {
```
Comment (gitkoshelew) [inline src/axios/axiosInstance.ts]: use ?. operator here

File: src/axios/axiosInstance.ts [hunk 2]
```diff
@@ -42,11 +43,13 @@ axiosInstance.interceptors.response.use(
 
     const { data } = response;
 
-    if (config.url === ENDPOINTS.refresh && data?.error?.statusCode === 401) {
+    if (config.url.includes('/refresh') && data?.error?.statusCode === 401) {
```
Comment (gitkoshelew) [inline src/axios/axiosInstance.ts]: leave ENDPOINTS.refresh variable here

File: src/screens/profile/createAccomodation/AccommodationImage/AccommodationImage.tsx [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+        {selectedImages.map((item, index) => (
+          <View key={index} style={styles.imageContainer}>
+            <Image source={{ uri: item.uri }} style={styles.image} />
+            <TouchableOpacity style={styles.deleteButton} onPress={() => handleDeleteImage(index)}>
+              <Icon name={IconName.Delete} size={20} color={WHITE} iconSet="material" />
+            </TouchableOpacity>
+          </View>
+        ))}
+
+        {selectedImages.length !== 0 && (
```
Comment (gitkoshelew) [inline src/screens/profile/createAccomodation/AccommodationImage/AccommodationImage.tsx]: looks like you can wrap once from line 131 till 164

File: src/axios/axiosInstance.ts (hunk 1 above)
Comment (melievkh) [reply src/axios/axiosInstance.ts]: when I use ?. it says this operator is only for reading properties, not assignment
so I make it like
  if (config.headers) {
        config.headers['refresh-token'] = refreshToken;
      }

**Review by fasbrothers — APPROVED** [review]

**Review by Sinumi — APPROVED** [review]

**Review by mirsadikov — APPROVED** [review]

**Pull request merged**